euphonic/writers/phonon_website.py

```diff
@@ -0,0 +1,218 @@
+"""Export to JSON for phonon visualisation website"""
```

Comment:
I know it's throughout the document, but the one place I think the link to "phonon visualisation website" should be would be here. I understand that others may come along in which case I would recommend:
```
"""Export to JSON for phonon visualisation websites:

- website 1
...
"""
```